docker-compose.yml

```diff
@@ -22,23 +22,24 @@ services:
   kafka:
     restart: always
     image: wurstmeister/kafka:latest
+    ports:
+      - "9092:9092"
```

Comment:
I think it's better to only expose the REST port from the api gateway in order to isolate as much as possible the app's environment. Maybe to expose them through the development `docker-compose` file would be better.

Reply:
In my opinion, since this is a test project I would leave it so for easier testing. For a production environment a certainly agree with you.

Reply:
Well, as this is a compromised topic, we can let the ports exposed by the moment.